lib/jake/integer.ex

```diff
@@ -1,7 +1,26 @@
 defmodule Jake.Integer do
   def gen(spec) do
-    min = Map.get(spec, "minimum", -9_007_199_254_740_991)
-    max = Map.get(spec, "maximum", 9_007_199_254_740_991)
-    StreamData.integer(min..max)
+    min = Map.get(spec, "minimum")
+    max = Map.get(spec, "maximum")
+    exclusive_min = Map.get(spec, "exclusiveMinimum", false)
+    exclusive_max = Map.get(spec, "exclusiveMaximum", false)
+    multipleOf = Map.get(spec, "multipleOf", 1)
+
+    min_bound =
+      case {min, exclusive_min} do
+        {nil, false} -> -9_007_199_254_740_991
+        {min, true} -> min + 1
```

Comment:
what if min is nil? when you pattern match, you have to consider all possibilities.

Reply:
if min is nil and exclusive_min is present, then that schema is invalid as per spec